dune-project

```diff
@@ -1,6 +1,5 @@
 (lang dune 2.7)
 (name opam-dune-lint)
-(formatting disabled)
```

Comment:
Why did you remove this? This project isn't using ocamlformat.

Reply:
True, is your concern the project is intended to not use ocamlformat or this has to be addressed in another PR ?

Reply:
The project does not intend to use ocamlformat, which is why the line was there.

(If a new maintainer takes over this project, they may decide to change the policy, but we can't have everyone who contributes a fix to a project also changing the rest of the project to their own preferred style.)

Reply:
I get it, it was not my intention to change the policy and the style I get it from solver-service https://github.com/ocurrent/solver-service/blob/main/.ocamlformat.